src/microbe_stage/gui/HUDMessages.tscn

```diff
@@ -0,0 +1,13 @@
+[gd_scene load_steps=3 format=2]
+
+[ext_resource path="res://src/microbe_stage/gui/HUDMessages.cs" type="Script" id=1]
+[ext_resource path="res://src/gui_common/fonts/Lato-Bold-Small.tres" type="DynamicFont" id=2]
+
+[node name="HUDMessages" type="VBoxContainer"]
+pause_mode = 1
+anchor_right = 1.0
+anchor_bottom = 1.0
```

Comment:
Should be anchored to the center top.

Reply:
Well I specifically anchored it on the whole screen as it kind of makes sense that the HUD messages can get extra space.

Reply:
Okay, either one works I guess.